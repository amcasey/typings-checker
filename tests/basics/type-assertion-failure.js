// This contains a type assertion which matches.
var v;
// $ExpectType string
v;
