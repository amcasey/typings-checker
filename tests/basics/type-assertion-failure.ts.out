tests/basics/type-assertion-failure.ts:3: Expected type
  string
but got:
  number

tests/basics/type-assertion-failure.ts: 0 / 1 checks passed.
